components/Nav/Nav.tsx

```diff
@@ -10,18 +10,19 @@ import {
   DrawerTitle,
   DrawerTrigger,
 } from '../NavDrawer/NavDrawer';
-import { logoutAccount } from '../../api/apiFunctions';
 import { useRouter } from 'next/navigation';
 import { usePathname } from 'next/navigation';
 import { cn } from '@/lib/utils';
+import { useAuthContext } from '@/context/AuthContextProvider';
 
 export const Nav = () => {
   const router = useRouter();
   const pathname = usePathname();
+  const { logoutAccount } = useAuthContext();
 
   const handleLogout = async () => {
     try {
-      await logoutAccount();
```

Comment:
Why was this await removed?

Reply:
the await isn't needed.  The user will ultimately be logged out.  so it just speeds it up.

wrong way to do it?

Reply:
It should be needed because if the function hangs up for any reason, it won't redirect the login prematurely.

Reply:
cool.   I can make that fix.